git-ssh.md

```diff
@@ -0,0 +1,75 @@
+# Installing SSH Key on Macbook for GitHub access
+
+I'm not going to walk through the reasoning behind using SSH keys as authorization or why GitHub is making this change. Let's just dive into the process and get the SSH keys created, installed, and referenced on GitHub.
+
+## Create the SSH Key
+1. Open terminal. **I recommend using the terminal application on your MacBook rather than the integrated terminal in VS Code.**
+1. Paste the following code into the terminal, substituting in your GitHub account email address for the `[your GitHub email]`:
+  ```
+  $ ssh-keygen -t -rsa -b 4096 -C "[your GitHub email]"
+  ```
+3. This generates a new SSH key, using the provided email as a label.
+  ```
+  > Generating public/private [your GitHub email] key pair.
+  ```
+4. When your promted to "Enter a file in which to save the key," press `Enter`. This accepts the default location.
+```
+> Enter a file in which to save the key (/Users/you/.ssh/id_youremailaddress):
```

Comment:
Default here is `/Users/username/.ssh/id_ed25519`
This file name is referenced later, so we should make sure this is consistent.